name.abuchen.portfolio.bootstrap/Application.e4xmi

```diff
@@ -86,7 +86,7 @@
       </children>
       <children xsi:type="menu:Menu" xmi:id="_y4b2QH0xEemaIIhNHihfSA" elementId="name.abuchen.portfolio.ui.menu.view" label="%menu.view.label">
         <children xsi:type="menu:Menu" xmi:id="_UnfJ8JkBEemdcMPaSHoNPw" elementId="name.abuchen.portfolio.ui.menu.view.options" label="%menu.view.options.label">
-          <children xsi:type="menu:HandledMenuItem" xmi:id="_C6CQwJkBEemdcMPaSHoNPw" elementId="name.abuchen.portfolio.ui.menu.view.start-with-current" label="Start with current view" type="Check" command="_kXGdcJkBEemdcMPaSHoNPw"/>
+          <children xsi:type="menu:HandledMenuItem" xmi:id="_C6CQwJkBEemdcMPaSHoNPw" elementId="name.abuchen.portfolio.ui.menu.view.start-with-current" label="%command.openWithLastView.name" type="Check" command="_kXGdcJkBEemdcMPaSHoNPw"/>
```

Comment:
The label is not actually used in the code. Because the label is replaced with the actual name of the current view to be configured (say "Statement of Assets"). See https://github.com/portfolio-performance/portfolio/blob/master/name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/handlers/SetCurrentViewAsInitialViewHandler.java#L33

This find this change a little confusing because it now uses the label for the similar but different functionality to use the "open with the view that was open when the file was open the last time". (also maybe better to be named but currently the open-with-last-view ;-))